chips/internal/trailing-icons.ts

```diff
@@ -34,10 +34,14 @@ export function renderRemoveButton({
       @focus=${focusListener}>
       <md-focus-ring part="trailing-focus-ring"></md-focus-ring>
       <md-ripple ?disabled=${disabled}></md-ripple>
-      <svg class="trailing icon" viewBox="0 96 960 960" aria-hidden="true">
-        <path
-          d="m249 849-42-42 231-231-231-231 42-42 231 231 231-231 42 42-231 231 231 231-42 42-231-231-231 231Z" />
-      </svg>
+      <span class="trailing icon">
```

Comment:
this change is 🤌

Reply:
\ o / I like how it reverted to just that